content/blog/2018/11/2018-11-12-inspecting-binaries-with-jenkins.adoc

```diff
@@ -0,0 +1,271 @@
+---
+layout: post
+title: "The Silence of the Lambs: Inspecting binaries, with Jenkins."
+tags:
+- devops
+- devsecops
+- security
+- vulnerabilities
+- compliance
+- twistlock
+author: michaelhuettermann
+---
+
+[NOTE]
```

Comment:
Now that we have an author section (thanks again @oleg-nenashev), maybe we can do less information here.